html/janus.js

```diff
@@ -1800,9 +1800,7 @@ function Janus(gatewayCallbacks) {
 				// For Chrome versions before 72, we force a plan-b semantic, and unified-plan otherwise
 				pc_config["sdpSemantics"] = (Janus.webRTCAdapter.browserDetails.version < 72) ? "plan-b" : "unified-plan";
 			}
-			var pc_constraints = {
-				"optional": [{"DtlsSrtpKeyAgreement": true}]
-			};
+			var pc_constraints = {};
 			if(ipv6Support) {
 				pc_constraints.optional.push({"googIPv6":true});
```

Comment:
Yes, delete this? googIPv6:true has been a no-op because IPv6 has been enabled by default for several years. So the removal of googIPv6 in M108 would only affect googIPv6:false use cases

Reply:
Sure, no problem, I'm ok with getting rid of that too! I think it's been there from the very start when it ad a purpose, and we forgot it there...